AudioKitSynthOne/DSP/Sequencer/S1Arpeggiator.hpp

```diff
@@ -42,7 +43,8 @@ struct Arpeggiator {
 
                 NoteNumber& note = sequencerNotes2[i];
                 const int nn = note.noteNumber + (octave * interval);
-                SeqNoteNumber snn{nn,1};
+                const int velocity = note.velocity;
+                SeqNoteNumber snn{nn, 1, velocity};
```

Comment:
Theoretically this could be inline as well (calling `note.velocity` in the initializer list)... but the compiler might optimize that for you anyway so no real need to change it